scheduler/plugin/plugin.go

```diff
@@ -187,6 +208,48 @@ func (pl *wasmPlugin) Name() string {
 	return PluginName
 }
 
+var _ framework.EnqueueExtensions = (*wasmPlugin)(nil)
+
+// allClusterEvents is copied from framework.go, to avoid the complexity of
+// conditionally implementing framework.EnqueueExtensions.
+var allClusterEvents = []framework.ClusterEvent{
+	{Resource: framework.Pod, ActionType: framework.All},
+	{Resource: framework.Node, ActionType: framework.All},
+	{Resource: framework.CSINode, ActionType: framework.All},
+	{Resource: framework.PersistentVolume, ActionType: framework.All},
+	{Resource: framework.PersistentVolumeClaim, ActionType: framework.All},
+	{Resource: framework.StorageClass, ActionType: framework.All},
+}
+
+// EventsToRegister implements the same method as documented on framework.EnqueueExtensions.
+func (pl *wasmPlugin) EventsToRegister() (clusterEvents []framework.ClusterEvent) {
```

Comment:
to make things stop explosion, we add a default impl for enqueue extensions with the same behavior as if it wasn't if the guest doesn't support them